modules/controller_pool/controller-primary.tpl

```diff
@@ -98,6 +98,23 @@ data:
       addresses:
       - ${metal_network_cidr}
 EOF
+
+  echo "Applying MetalLB manifests..." && \
+    cd $HOME/kube && \
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f $(cat $HOME/workloads.json | jq .metallb_namespace) && \
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f $(cat $HOME/workloads.json | jq .metallb_release) && \
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)" && \
+    kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f metal_lb.yaml
+}
+
+function kube_vip {
+  alias kube-vip="docker run --network host --rm ghcr.io/kube-vip/kube-vip:0.3.8"
+  kube-vip manifest daemonset \
+  --interface $INTERFACE \
```

Comment:
@displague This env var comes from the kube-vip [docs](https://kube-vip.io/usage/EquinixMetal/#using-annotations); any idea where I can grab this when building out the cluster?

Reply:
Looking at. https://kube-vip.io/hybrid/daemonset/#bgp-example, and other examples specific to Equinix Metal, it seems `lo` is the interface to use.

Sound right, @thebsdbox?